CrossrefReferenceLinkingInfoSender.php

```diff
@@ -0,0 +1,79 @@
+<?php
+
+/**
+ * @file plugins/generic/crossrefReferenceLinking/CrossrefReferenceLinkingInfoSender.php
+ *
+ * Copyright (c) 2013-2023 Simon Fraser University
+ * Copyright (c) 2003-2023 John Willinsky
+ * Distributed under the GNU GPL v3. For full terms see the file LICENSE.
+ *
+ * @class CrossrefReferenceLinkingInfoSender
+ * @brief Scheduled task to check for found Crossref references DOIs.
+ */
+
+namespace APP\plugins\generic\crossrefReferenceLinking;
+
+use APP\core\Application;
+use PKP\plugins\PluginRegistry;
+use PKP\scheduledTask\ScheduledTask;
+
+class CrossrefReferenceLinkingInfoSender extends ScheduledTask
+{
+
+	protected CrossrefReferenceLinkingPlugin $plugin;
+
+	/**
+	 * Constructor.
+	 * @param $args array task arguments
+	 */
+	public function __construct($args)
+	{
+		$this->plugin = PluginRegistry::getPlugin('generic', 'crossrefreferencelinkingplugin');
+		$this->plugin->addLocaleData();
+		parent::__construct($args);
```

Comment:
I think it's generally better to move this to the first line, but not a big deal 😁